File: EulerProject/0xx/00x/006/main.cs
using System;
using System.Collections.Generic;

class Program {
    public static void Main () {
        ulong sum1 = 0, sum2 = 0;
        for (ulong i = 0; i <= 100; i++) {
            sum1 += i * i;
            sum2 += i;
        }
        sum2 *= sum2;
        Console.WriteLine(sum2 - sum1);
    }
}

File: EulerProject/0xx/00x/006/main.c
#include <stdio.h>
int main() {
    long long sum1 = 0, sum2 = 0;
    for (int i = 0; i <= 100; i++) {
        sum1 += i * i;
        sum2 += i;
    }
    sum2 *= sum2;
    printf ("%lld", sum2 - sum1);
}